etabotsite/etabotapp/models.py

```diff
@@ -200,3 +236,20 @@ def parse_projects_for_TMS(instance, **kwargs):
  Updated existing projects: {}".format(
         ', '.join(new_projects),
         ', '.join(updted_projects))
+
+
+from django.conf import settings
```

Comment:
from PEP8 python code style guideline, it's better to keep all the imports at one place, preferably at the beginning of a python file.

Reply:
moved